commands/diff.go

```diff
@@ -4,13 +4,15 @@ import (
 	"io"
 	"os"
 
+	"github.com/10gen/realm-cli/utils/telemetry"
+
 	"github.com/10gen/realm-cli/models"
 	"github.com/10gen/realm-cli/utils"
 	"github.com/mitchellh/cli"
 )
 
 // NewDiffCommandFactory returns a new cli.CommandFactory given a cli.Ui
-func NewDiffCommandFactory(ui cli.Ui) cli.CommandFactory {
+func NewDiffCommandFactory(ui cli.Ui, service *telemetry.Service) cli.CommandFactory {
```

Comment:
when outside the `telemetry` package, lets make sure to refer to this as `telemetryService` as opposed to just a generic `service`